runtime/lua/vim/shared.lua

```diff
@@ -4,34 +4,36 @@
 -- test-suite. If, in the future, Nvim itself is used to run the test-suite
 -- instead of "vanilla Lua", these functions could move to src/nvim/lua/vim.lua
 
+local shared = {}
```

Comment:
shared.lua is a special case, we actually consider it part of the `vim` module. So I think this should be `vim` -- that will help ctags, for example.

We won't ever need to `require('vim')` in this module (because this module, by definition, is pure Lua--without Nvim), so there won't be any conflict here.

Reply:
OK. BTW when I first read Neovim code, I was a little confused by the file name shared.lua.
Although it is a module shared with test helper, util.lua seems to be easier to understand, what do you think?

Reply:
why would that be easier to understand?

Reply:
Maybe because of my little experience.

Reply:
> But that module test file nema is `utility_funcitons_spec.lua`.

Yeah, I don't know what that test is named so. We should rename it to `vim_spec.lua`